<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindObjectDlg</class>
 <widget class="QDialog" name="FindObjectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Objects</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Find Information</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Slot List</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mSlotsCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slot Status</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QLineEdit" name="mSessionText">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Session</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mSlotIDText">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mLoginText">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Slot ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QLineEdit" name="mClassText">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="mKeyTypeLabel">
       <property name="text">
        <string>Key Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mClassCombo"/>
     </item>
     <item row="0" column="8">
      <widget class="QLineEdit" name="mMaxText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="mKeyTypeCombo"/>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="mKeyTypeText">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>221</red>
             <green>221</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled/>
        </palette>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mFirstTab">
      <attribute name="title">
       <string>Information</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="2" column="0">
          <widget class="QLabel" name="mSubjectLabel">
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mSubjectText"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="mLabelLabel">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mLabelText"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mIDLabel">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="mPubKeyInfoText"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="mApplicationText"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="mPubKeyInfoLabel">
           <property name="text">
            <string>PubKeyInfo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mIDText"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="mApplicationLabel">
           <property name="text">
            <string>Application</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="mObjectIDLabel">
           <property name="text">
            <string>Object ID</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="mObjectIDText"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mSecondTab">
      <attribute name="title">
       <string>Boolean</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="4">
          <widget class="QCheckBox" name="mWrapCheck">
           <property name="text">
            <string>Wrap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="mEncryptCheck">
           <property name="text">
            <string>Encrypt</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="mModifiableCombo"/>
         </item>
         <item row="2" column="5">
          <widget class="QComboBox" name="mVerifyCombo"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="mUnwrapCombo"/>
         </item>
         <item row="3" column="5">
          <widget class="QComboBox" name="mDeriveCombo"/>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="mEndDateCheck">
           <property name="text">
            <string>EndDate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="mPrivateCombo"/>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="mWrapCombo"/>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="mModifiableCheck">
           <property name="text">
            <string>Modifiable</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="mUnwrapCheck">
           <property name="text">
            <string>Unwrap</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QComboBox" name="mTrustedCombo"/>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="mStartDateCheck">
           <property name="text">
            <string>StartDate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="mSensitiveCheck">
           <property name="text">
            <string>Sensitive</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="mEncryptCombo"/>
         </item>
         <item row="5" column="3">
          <widget class="QComboBox" name="mDestroyableCombo"/>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="mExtractableCheck">
           <property name="text">
            <string>Extractable</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="mDestroyableCheck">
           <property name="text">
            <string>Destroyable</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="mVerifyCheck">
           <property name="text">
            <string>Verify</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="mCopyableCombo"/>
         </item>
         <item row="1" column="5">
          <widget class="QComboBox" name="mDecryptCombo"/>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="mSensitiveCombo"/>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="mPrivateCheck">
           <property name="text">
            <string>Private</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="mCopyableCheck">
           <property name="text">
            <string>Copyable</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="mSignCheck">
           <property name="text">
            <string>Sign</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="mTrustedCheck">
           <property name="text">
            <string>Trusted</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="mDecryptCheck">
           <property name="text">
            <string>Decrypt</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="mTokenCombo"/>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="mDeriveCheck">
           <property name="text">
            <string>Derive</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="mSignCombo"/>
         </item>
         <item row="3" column="3">
          <widget class="QComboBox" name="mExtractableCombo"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="mTokenCheck">
           <property name="text">
            <string>Token</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDateEdit" name="mStartDateEdit"/>
         </item>
         <item row="4" column="3">
          <widget class="QDateEdit" name="mEndDateEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="3">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mFindObjectsBtn">
       <property name="text">
        <string>Find Objets</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSlotsCombo</tabstop>
  <tabstop>mSlotIDText</tabstop>
  <tabstop>mSessionText</tabstop>
  <tabstop>mLoginText</tabstop>
  <tabstop>mClassCombo</tabstop>
  <tabstop>mKeyTypeCombo</tabstop>
  <tabstop>mMaxText</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>mLabelText</tabstop>
  <tabstop>mIDText</tabstop>
  <tabstop>mSubjectText</tabstop>
  <tabstop>mPubKeyInfoText</tabstop>
  <tabstop>mApplicationText</tabstop>
  <tabstop>mObjectIDText</tabstop>
  <tabstop>mPrivateCheck</tabstop>
  <tabstop>mPrivateCombo</tabstop>
  <tabstop>mSensitiveCheck</tabstop>
  <tabstop>mSensitiveCombo</tabstop>
  <tabstop>mWrapCheck</tabstop>
  <tabstop>mWrapCombo</tabstop>
  <tabstop>mUnwrapCheck</tabstop>
  <tabstop>mUnwrapCombo</tabstop>
  <tabstop>mEncryptCheck</tabstop>
  <tabstop>mEncryptCombo</tabstop>
  <tabstop>mDecryptCheck</tabstop>
  <tabstop>mDecryptCombo</tabstop>
  <tabstop>mModifiableCheck</tabstop>
  <tabstop>mModifiableCombo</tabstop>
  <tabstop>mSignCheck</tabstop>
  <tabstop>mSignCombo</tabstop>
  <tabstop>mVerifyCheck</tabstop>
  <tabstop>mVerifyCombo</tabstop>
  <tabstop>mTokenCheck</tabstop>
  <tabstop>mTokenCombo</tabstop>
  <tabstop>mExtractableCheck</tabstop>
  <tabstop>mExtractableCombo</tabstop>
  <tabstop>mDeriveCheck</tabstop>
  <tabstop>mDeriveCombo</tabstop>
  <tabstop>mStartDateCheck</tabstop>
  <tabstop>mStartDateEdit</tabstop>
  <tabstop>mEndDateCheck</tabstop>
  <tabstop>mEndDateEdit</tabstop>
  <tabstop>mTrustedCheck</tabstop>
  <tabstop>mTrustedCombo</tabstop>
  <tabstop>mCopyableCheck</tabstop>
  <tabstop>mCopyableCombo</tabstop>
  <tabstop>mDestroyableCheck</tabstop>
  <tabstop>mDestroyableCombo</tabstop>
  <tabstop>mFindObjectsBtn</tabstop>
  <tabstop>mCloseBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
